### PR TITLE
Update user map for July 2017 #7898

### LINKED ISSUE #7898: Update user map for July 2017
Similar to #7215 
New data to consider:

Country | Institute
-- | --
Australia | Australian National University
Australia | St Paul's Anglican Grammar School
Brazil | Universidade Estadual de Maringá
Canada | Dalhousie
Canada | Sheridan College
Canada | University of Guelph
Denmark | DHI
El Salvador | Escuela Superior de Economía y Negocios
France | EMLYON Business School
Germany | Stuttgart Media University
India | KBP COE SATARA
Kazakhstan | Method Coding School
Mexico | UNAM
Netherlands | Avans University of Applied Sciences
Netherlands | Eindhoven University of Technology
New Zealand | Massey University
New Zealand | University of Auckland
Pakistan | Aga Khan University Hospital
Philippines | EKR
Puerto Rico | Carlos Albizu University
Singapore | Australian International School Singapore
Singapore | River Valley High School
Singapore | Singapore Institute of Technology
Singapore | Singapore Polytechnic
Singapore | sport.gov.sg
Spain | Geneva Business School Barcelona Campus
Taiwan | NKFUST
UK | Hult International Business School
UK | Kingston University
UK | Liverpool John Moores University
UK | Newcastle University
UK | University of Huddersfield
UK | University of Portsmouth
UK | University of Warwick
USA | Amazon
USA | CalBaptist University
USA | Carly Thompson Temp University
USA | Claremont Colleges
USA | Cotton Creek Elementary
USA | Dayton Regional STEM School
USA | Duke University
USA | Jacksonville Christian Academy
USA | Keiser University
USA | Kendall College of Art and Design
USA | Lake Sumter State College
USA | Middle Creek High School
USA | Northeastern University
USA | Ohio University
USA | Penn State University
USA | Quinnipiac University
USA | Southern Connecticut State University
USA | Southern Oregon University
USA | St. Magarets School
USA | The University of Memphis
USA | University of Cincinnati
USA | University of Dallas
USA | University of South Carolina Aiken
USA | University of South Florida
USA | Van Buren Elementary
USA | Volusia Online Learning
USA | Wellesley College
USA | Xavier University of Louisiana

### PR DESCRIPTION
**Pull request by sowmyadvn**
Fixes #7898 
Added new universities to userMapData.json
Haven't added EKR, Philippines. After thorough google search, I have found that there is no university with the acronym. Please correct me if I am wrong.

### CODE REVIEW

File: src/main/webapp/data/userMapData.json [hunk 1]
```diff
@@ -48,6 +48,7 @@
       "RMIT University",
       "St Catherine's Catholic College",
       "St Ives High School",
+       "St Paul's Anglican Grammar School",
```
Comment (whipermr5) [inline src/main/webapp/data/userMapData.json]: Fix the indentation here.

File: src/main/webapp/data/userMapData.json [hunk 2]
```diff
@@ -1,5 +1,5 @@
 {
-  "lastUpdated": "End of June 2017",
+  "lastUpdated": "5th September 2017",
```
Comment (whipermr5) [inline src/main/webapp/data/userMapData.json]: Use `End of July 2017`

Comment (sowmyadvn) [reply src/main/webapp/data/userMapData.json]: Updated to "End of July 2017"

File: src/main/webapp/data/userMapData.json (hunk 1 above)
Comment (sowmyadvn) [reply src/main/webapp/data/userMapData.json]: @whipermr5 Fixed. Thanks for reviewing my PR. 👍

File: src/main/webapp/data/userMapData.json [hunk 3]
```diff
@@ -1345,6 +1375,7 @@
       "North Carolina School of Science and Mathematics",
       "North Carolina State University",
       "North Central College",
+      "Northeastern University",
```
Comment (whipermr5) [inline src/main/webapp/data/userMapData.json]: There's already an entry for this 3 lines below.

File: src/main/webapp/data/userMapData.json [hunk 4]
```diff
@@ -1238,6 +1266,7 @@
       "Kit Carson World School",
       "Knox High School",
       "Lakeridge Junior High",
+      "Lake Sumter State College",
```
Comment (whipermr5) [inline src/main/webapp/data/userMapData.json]: This should come before `Lakeridge Junior High` - shorter words should come before longer words in alphabetical ordering.

File: src/main/webapp/data/userMapData.json [hunk 5]
```diff
@@ -1286,6 +1315,7 @@
       "Metropolitan Community College",
       "MGH Institute of Health Professions",
       "Mid Pacific Institute",
+      "Middle Creek High School",
       "Miami Dade College",
       "Miami International University of Art and Design",
       "Miami University",
```
Comment (whipermr5) [inline src/main/webapp/data/userMapData.json]: Not your fault, but I found out that `Miami University` and `Miami University of Ohio` (one line below) are the same university. Could you delete `Miami University of Ohio`?

Comment (sowmyadvn) [reply src/main/webapp/data/userMapData.json]: Deleted it.

File: src/main/webapp/data/userMapData.json (hunk 3 above)
Comment (sowmyadvn) [reply src/main/webapp/data/userMapData.json]: Removed the duplicate entry. 👍

File: src/main/webapp/data/userMapData.json (hunk 4 above)
Comment (sowmyadvn) [reply src/main/webapp/data/userMapData.json]: Thanks for correcting. Changed accordingly.

File: src/main/webapp/data/userMapData.json (hunk 3 above)
Comment (whipermr5) [reply src/main/webapp/data/userMapData.json]: The original entry was at the correct position. `Northeastern` should come after `North`.

**Pull request merged**